modules/standard/Math.chpl

```diff
@@ -1057,6 +1057,66 @@ module Math {
     return truncf(x);
   }
 
+
+  /* Returns the Bessel function of the first kind of order `0` of `x`. */
+  extern proc j0(x: real(64)): real(64);
+
+  /* Returns the Bessel function of the first kind of order `0` of `x`. */
+  inline proc j0(x : real(32)): real(32) {
+    extern proc j0f(x: real(32)): real(32);
+    return j0f(x);
+  }
+
+
+  /* Returns the Bessel function of the first kind of order `1` of `x`. */
+  extern proc j1(x: real(64)): real(64);
+
+  /* Returns the Bessel function of the first kind of order `1` of `x`. */
+  inline proc j1(x : real(32)): real(32) {
+    extern proc j1f(x: real(32)): real(32);
+    return j1f(x);
+  }
+
+
+  /* Returns the Bessel function of the first kind of order `n` of `x`. */
+  extern proc jn(n: int, x: real(64)): real(64);
```

Comment:
Note, Chapel int != C int; Chapel int corresponds to int64_t.
It would be possible to use c_int in the argument but a better practice is probably to use safeCast.

Reply:
OK, thanks I make changes, I didn't thought of that, that time, thanks

Reply:
Its done

Reply:
I think you've correctly updated the 'extern proc' but I think the Chapel versions should use `int` as the argument type and then use `safeCast` to convert from int to c_int.